### ISSUE TITLE
Module Build failed

### ISSUE THREAD

**Issue by zexingguo** [adeptoas/sweet-modal-vue#29]
Module build failed: Error: Couldn't find preset "es2015" relative to directory. 

When I important the component it shows me module bild failed, it cant find the .
babelrc in sweet-modal-vue

**Comment by saybabi** [comment]
try to install babel-preset-es2015

**Comment by smpl** [comment]
i have same problem
step to reproduce
```
vue-cli webpack
npm install sweet-modal-vue --save-dev
npm install babel-preset-es2015 --save
```
add import { SweetModal, SweetModalTab } from 'sweet-modal-vue'
and i have problem with es2015

**Comment by bluefirex** [comment]
We'll be switching away from `es2015` to `env` in the next release.

**Comment by bluefirex** [comment]
Fixed in c05fb535b10cb39d2bfa17c218d198471a1e37da, released in [v2.0.0](https://github.com/adeptoas/sweet-modal-vue/releases/tag/v2.0.0)
